Assignment6_docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:latest
+MAINTAINER Nishu Singh, nsingh2878@sdsu.edu
+
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+
+RUN mkdir ./sql
+RUN mkdir ./script
+RUN mkdir ./output
+
+RUN ["apt-get", "update"]
+RUN ["apt-get", "install", "-y", "mariadb-client"]
+
```

Comment:
Not sure why some RUN commands were using [ ] but some were not. Maybe it's still better to stay with one style even if they are equivalent.